tests/ui/test_views.py

```diff
@@ -24,13 +24,13 @@
 
 """Views tests."""
 
-from __future__ import absolute_import, print_function
```

Comment:
Why?

Reply:
It's from the cookiecutter, You have to ask  invenio.